Bottom: 180f7c1c55f5decd8030de457d82d654c2d88073
Top:    8ab05755f359a7795fcb7a5a793866cad03d1855
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 16:25:36 +0000

Refresh of toporouter-comment-unused-vari

---

diff --git a/src/toporouter.c b/src/toporouter.c
index 0ad26b5..41fd4b1 100644
--- a/src/toporouter.c
+++ b/src/toporouter.c
@@ -2577,11 +2577,10 @@ check_cons_continuation:
   }
   i = l->constraints;
   while (i) {
-    
-    toporouter_constraint_t *c1 = TOPOROUTER_CONSTRAINT(i->data);
-    
-  //  printf("adding cons: "); print_constraint(c1);
-    
+
+    // toporouter_constraint_t *c1 = TOPOROUTER_CONSTRAINT(i->data);
+    // printf("adding cons: "); print_constraint(c1);
+
     GSList *conflicts = gts_delaunay_add_constraint (l->surface, i->data);
     GSList *j = conflicts;
     while(j) {
